motivaa-backend/motivaa-app/src/main/java/com/motivaa/control/repository/Elasticsearch.java

```diff
@@ -55,6 +54,7 @@ public List<User> searchAllUsers() throws java.io.IOException {
                 SearchResponse<User> searchResponse = esClient.search(s->s
                         .index("user"),User.class);
                 List<Hit<User>> hits = searchResponse.hits().hits();
+                log.info(searchResponse.hits().hits());
```

Comment:
you could do this too: log.info(hits) so you don't compute the hits twice.
Also maybe you could add some context into the log, not just printing the hits.

Reply:
btw it is not very clear to me why you added a log here and not below to the new method.

Reply:
done